test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -0,0 +1,375 @@
+
+const {
+  convertToHexValue,
+  withFixtures,
+  connectDappWithExtensionPopup,
+  getWindowHandles,
+} = require('../helpers');
+
+describe('Create token, approve token and approve token without gas', function () {
+  describe('Add a custom token from a dapp', function () {
+    const ganacheOptions = {
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: convertToHexValue(25000000000000000000),
+        },
+      ],
+    };
+
+    it('creates, imports and renders the balance for the new token', async function () {
+      await withFixtures(
+        {
+          dapp: true,
+          fixtures: 'imported-account',
+          ganacheOptions,
+          title: this.test.title,
+        },
+        async ({ driver }) => {
+          await driver.navigate();
+          await driver.fill('#password', 'correct horse battery staple');
+          await driver.press('#password', driver.Key.ENTER);
+
+          await connectDappWithExtensionPopup(driver);
+          // create token
+          await driver.waitForSelector({ text: 'Create Token', tag: 'button' });
+          await driver.clickElement({ text: 'Create Token', tag: 'button' });
+
+          const windowHandles = await getWindowHandles(driver, 3);
+          await driver.switchToWindow(windowHandles.popup);
+          await driver.clickElement({ text: 'Edit', tag: 'button' });
+          const inputs = await driver.findElements('input[type="number"]');
+          const gasLimitInput = inputs[0];
+          const gasPriceInput = inputs[1];
+          await gasLimitInput.fill('4700000');
+          await gasPriceInput.fill('20');
+          await driver.waitForSelector({ text: 'Save', tag: 'button' });
+          await driver.clickElement({ text: 'Save', tag: 'button' });
+          await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+          await driver.switchToWindow(windowHandles.dapp);
+
+          const tokenContractAddress = await driver.waitForSelector({
+            css: '#tokenAddress',
+            text: '0x',
+          });
+          const tokenAddress = await tokenContractAddress.getText();
+
+          // imports custom token from extension
+          await driver.switchToWindow(windowHandles.extension);
+          await driver.clickElement(`[data-testid="home__asset-tab"]`);
+          await driver.clickElement({ tag: 'button', text: 'Assets' });
+
+          await driver.clickElement({ text: 'import tokens', tag: 'a' });
+          await driver.clickElement({
+            text: 'Custom Token',
+            tag: 'button',
+          });
+          await driver.waitForSelector('#custom-address');
+          await driver.fill('#custom-address', tokenAddress);
+          await driver.waitForSelector('#custom-symbol');
+          await driver.waitForSelector('#custom-decimals');
+          const addButton = await driver.findClickableElement({
```

Comment:
can we just use `clickElement` here, as it calls `findClickableElement` under the hood? It doesn't look like addButton is used anywhere else